### DISCUSSION TITLE
should non-existent /var/lib/vector/ be created automatically

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#11390] [Q&A] [answered]
I get this when I run vector on fresh manual install:

> data_dir "/var/lib/vector/" does not exist

I expected this to be created automatically. I suspect the other installs (like yum and apt) do create such?

**Comment by jszwedko** [comment, marked as answer]
Hey @tshepang !

Yeah, that is correct, the OS packages do create that directory. I agree with you though, we could have Vector try to create the directory if it does not already exist.
